startup/BMM/workspace.py

```diff
@@ -4,10 +4,15 @@
 import redis
 from BMM.functions import verbosebold_msg, error_msg
 
+if not os.environ.get('AZURE_TESTING'):
+    redis_host = 'xf06bm-ioc2'
+else:
+    redis_host = '127.0.0.1'
```

Comment:
@bruceravel, we need to parametrize this redis host address as in the CI system we can't use NSLS-II specific hostnames easily. At BMM, you are not going to have the `AZURE_TESTING` environment variable set up, so the hostname will be `xf06bm-ioc2`. For CI. we configure it to be `127.0.0.1`, as it's the default address after installation.